src/modules/clearcm.js

```diff
@@ -22,7 +22,4 @@ export const clearAllTask = () => {
   });
   task.push(...newArr);
   localStorage.setItem('task', JSON.stringify(task));
```

Comment:
- [ ] I am not able to edit a to do please work on the `edit` method
- [ ] Make sure the property `index` should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).
- [ ] When you delete a task make sure that it update all remaining items' indexes